draft-ietf-rats-corim.md

```diff
@@ -1118,6 +1118,21 @@ are acceptable states.
 Integrity Registers can be used to model the PCRs in a TPM or vTPM, in which case the identifier is the register index, or other kinds of vendor-specific measured objects.
 
 
+##### Linear Privilege Level {#sec-comid-linear-privlevel}
+
+A Linear Privilege Level describes an operating privilege for the target environment.
```

Comment:
I would swap the bottom sentence to state that:

Reply:
Please clarify.